plugins/meta/bandwidth/bandwidth_linux_test.go

```diff
@@ -621,6 +621,257 @@ var _ = Describe("bandwidth test", func() {
 		})
 	})
 
+	Describe("get the host interface which plugin should work on", func() {
```

Comment:
Please look how `Describe` messages are used in other test cases, like how are used in `It`. It could be something like:
```go
Describe("Getting host interface from veth pair"), func() {
    It("Should work with multiple host veth interfaces", ...
```

Reply:
Thanks for review, updated.